python311Packages.shellingham: 1.5.0.post1 -> 1.5.1

fabaff opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python311Packages.shellingham: 1.5.0.post1 -> 1.5.1

Diff: https://github.com/sarugaku/shellingham/compare/refs/tags/1.5.0.post1...1.5.1

fabaff committed:
python311Packages.shellingham: add changelog to meta

fabaff commented:
This PR should go before #237113.

fabaff commented:
Result of `nixpkgs-review pr 237110` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>70 packages failed to build:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>home-assistant-component-tests.ovo_energy</li>
    <li>home-assistant-component-tests.unifiprotect</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>
<details>
  <summary>38 packages built:</summary>
  <ul>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>platformio</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
  </ul>
</details>

tjni commented:
Result of `nixpkgs-review pr 237110` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>napari</li>
    <li>napari.dist</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.napari.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
  </ul>
</details>
<details>
  <summary>10 packages failed to build:</summary>
  <ul>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
  </ul>
</details>
<details>
  <summary>89 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>

tjni reviewed (approved):
LGTM on aarch64-darwin.

`textnets` and `rstcheck` are already broken and `toggl-cli` failure did not happen when the build was run a second time.

fabaff commented:
`typer` fails now.

fabaff committed:
python311Packages.typer: tests are failing on Linux as well

fabaff commented:
Result of `nixpkgs-review pr 237110` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>70 packages failed to build:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>home-assistant-component-tests.ovo_energy</li>
    <li>home-assistant-component-tests.unifiprotect</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>
<details>
  <summary>38 packages built:</summary>
  <ul>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>platformio</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
  </ul>
</details>

fabaff commented:
Result of `nixpkgs-review pr 237110` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>pacup</li>
    <li>pacup.dist</li>
  </ul>
</details>
<details>
  <summary>118 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>home-assistant-component-tests.ovo_energy</li>
    <li>home-assistant-component-tests.unifiprotect</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>platformio</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.alexapy</li>
    <li>python310Packages.alexapy.dist</li>
    <li>python310Packages.argilla</li>
    <li>python310Packages.argilla.dist</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.alexapy</li>
    <li>python311Packages.alexapy.dist</li>
    <li>python311Packages.argilla</li>
    <li>python311Packages.argilla.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>

fabaff committed:
pacup: 1.1.0 -> 2.0.0

Diff: pacstall/pacup@refs/tags/2.0.0...2.0.0

Changelog: https://github.com/pacstall/pacup/releases/tag/2.0.0

fabaff commented:
Result of `nixpkgs-review pr 237110` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>120 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>home-assistant-component-tests.ovo_energy</li>
    <li>home-assistant-component-tests.unifiprotect</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>pacup</li>
    <li>pacup.dist</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>platformio</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.alexapy</li>
    <li>python310Packages.alexapy.dist</li>
    <li>python310Packages.argilla</li>
    <li>python310Packages.argilla.dist</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.alexapy</li>
    <li>python311Packages.alexapy.dist</li>
    <li>python311Packages.argilla</li>
    <li>python311Packages.argilla.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>

tjni commented:
Result of `nixpkgs-review pr 237110` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>napari</li>
    <li>napari.dist</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.napari-npe2</li>
    <li>python310Packages.napari-npe2.dist</li>
    <li>python310Packages.napari.dist</li>
    <li>python310Packages.torchio</li>
    <li>python310Packages.torchio.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>rstcheck (python310Packages.rstcheck)</li>
    <li>rstcheck.dist (python310Packages.rstcheck.dist)</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python311Packages.rstcheck</li>
    <li>python311Packages.rstcheck.dist</li>
  </ul>
</details>
<details>
  <summary>103 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>doitlive</li>
    <li>doitlive.dist</li>
    <li>esphome</li>
    <li>esphome.dist</li>
    <li>hatch</li>
    <li>hatch.dist</li>
    <li>markdown-anki-decks</li>
    <li>markdown-anki-decks.dist</li>
    <li>pacup</li>
    <li>pacup.dist</li>
    <li>pdm</li>
    <li>pdm.dist</li>
    <li>platformio-core</li>
    <li>platformio-core.dist</li>
    <li>platformio-core.udev</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python310Packages.alexapy</li>
    <li>python310Packages.alexapy.dist</li>
    <li>python310Packages.argilla</li>
    <li>python310Packages.argilla.dist</li>
    <li>python310Packages.authcaptureproxy</li>
    <li>python310Packages.authcaptureproxy.dist</li>
    <li>python310Packages.click-completion</li>
    <li>python310Packages.click-completion.dist</li>
    <li>python310Packages.coinmetrics-api-client</li>
    <li>python310Packages.coinmetrics-api-client.dist</li>
    <li>python310Packages.dsnap</li>
    <li>python310Packages.dsnap.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.gradient</li>
    <li>python310Packages.gradient.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
    <li>python310Packages.ovoenergy</li>
    <li>python310Packages.ovoenergy.dist</li>
    <li>python310Packages.pathy</li>
    <li>python310Packages.pathy.dist</li>
    <li>python310Packages.pipenv-poetry-migrate</li>
    <li>python310Packages.pipenv-poetry-migrate.dist</li>
    <li>python310Packages.pyunifiprotect</li>
    <li>python310Packages.pyunifiprotect.dist</li>
    <li>python310Packages.shellingham</li>
    <li>python310Packages.shellingham.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.teslajsonpy</li>
    <li>python310Packages.teslajsonpy.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.toggl-cli</li>
    <li>python310Packages.toggl-cli.dist</li>
    <li>python310Packages.typer</li>
    <li>python310Packages.typer.dist</li>
    <li>python310Packages.ward</li>
    <li>python310Packages.ward.dist</li>
    <li>python311Packages.alexapy</li>
    <li>python311Packages.alexapy.dist</li>
    <li>python311Packages.argilla</li>
    <li>python311Packages.argilla.dist</li>
    <li>python311Packages.authcaptureproxy</li>
    <li>python311Packages.authcaptureproxy.dist</li>
    <li>python311Packages.click-completion</li>
    <li>python311Packages.click-completion.dist</li>
    <li>python311Packages.coinmetrics-api-client</li>
    <li>python311Packages.coinmetrics-api-client.dist</li>
    <li>python311Packages.dsnap</li>
    <li>python311Packages.dsnap.dist</li>
    <li>python311Packages.gradient</li>
    <li>python311Packages.gradient.dist</li>
    <li>python311Packages.ovoenergy</li>
    <li>python311Packages.ovoenergy.dist</li>
    <li>python311Packages.pathy</li>
    <li>python311Packages.pathy.dist</li>
    <li>python311Packages.pipenv-poetry-migrate</li>
    <li>python311Packages.pipenv-poetry-migrate.dist</li>
    <li>python311Packages.pyunifiprotect</li>
    <li>python311Packages.pyunifiprotect.dist</li>
    <li>python311Packages.shellingham</li>
    <li>python311Packages.shellingham.dist</li>
    <li>python311Packages.teslajsonpy</li>
    <li>python311Packages.teslajsonpy.dist</li>
    <li>python311Packages.toggl-cli</li>
    <li>python311Packages.toggl-cli.dist</li>
    <li>python311Packages.typer</li>
    <li>python311Packages.typer.dist</li>
    <li>python311Packages.ward</li>
    <li>python311Packages.ward.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell_gpt</li>
    <li>shell_gpt.dist</li>
    <li>unifi-protect-backup</li>
    <li>unifi-protect-backup.dist</li>
  </ul>
</details>

tjni reviewed (approved):
All of the failed packages already fail on macOS. This change LGTM.
